dlc_developer_config.toml

```diff
@@ -31,13 +31,13 @@ eks_tests = true
 ec2_tests = true
```

Comment:
Is there a way such that we can add a config variable to run the tests on the PR even if they have been disabled for the PR context?

Reply:
Do you mean like a universal "on" switch? Currently the only things that are off are the sm local and sm remote tests. Since sm remote tests come in several different configurations, this could get confusing for users. I would suggest not adding this type of option for that reason.

Reply:
Sure. Thanks.